project_scrum/models/project_scrum_meeting.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api, _
+
+
+class ProjectScrumMeeting(models.Model):
+    _name = 'project.scrum.meeting'
+    _description = 'Project Scrum Daily Meetings'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='cascade',
+        index=True,
+        track_visibility='onchange',
+        change_default=True,
+        required=True,
+    )
+    sprint_id = fields.Many2one(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    datetime_meeting = fields.Datetime(
+        string='Date and Time of the meeting',
+        required=True,
+    )
+    user_id_meeting = fields.Many2one(
```

Comment:
You might change the string to "Meeting organiser" then you can keep it a Many2one, but it will be more clear why you can only select one person.